src/Libraries/Microsoft.Extensions.AI.Evaluation.Console/Commands/ReportCommand.cs

```diff
@@ -51,14 +51,25 @@ internal async Task<int> InvokeAsync(
 
         List<ScenarioRunResult> results = [];
 
+        string? firstExecutionName = null;
+
         await foreach (string executionName in
             resultStore.GetLatestExecutionNamesAsync(lastN, cancellationToken).ConfigureAwait(false))
         {
+            firstExecutionName ??= executionName;
```

Comment:
Consider renaming to `latestExecutionName` or `mostRecenetExecutionName` since first can be confused with the first one in the trends table (which would be the oldest).